.evergreen.yml

```
@@ -41,6 +41,40 @@ functions:
           GOROOT: ${go_root}
           <<: *go_options
         command: make test
+  "atlas e2e":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          WORKDIR: ${workdir}
+          GOROOT: ${go_root}
+          <<: *go_options
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_ORG_ID: ${mcli_org_id}
+          MCLI_PRIVATE_API_KEY: ${mcli_private_api_key}
+          MCLI_PROJECT_ID: ${mcli_project_id}
+          MCLI_PUBLIC_API_KEY: ${mcli_public_api_key}
```

small nit, let leave the append (`<<`)  to the end
```suggestion
          WORKDIR: ${workdir}
          GOROOT: ${go_root}
          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
          MCLI_ORG_ID: ${mcli_org_id}
          MCLI_PRIVATE_API_KEY: ${mcli_private_api_key}
          MCLI_PROJECT_ID: ${mcli_project_id}
          MCLI_PUBLIC_API_KEY: ${mcli_public_api_key}
          <<: *go_options
```